src/Compilers/Core/MSBuildTask/project.json

```diff
@@ -12,6 +12,7 @@
         "System.IO.FileSystem": "4.0.1",
         "System.IO.FileSystem.DriveInfo": "4.0.0",
         "System.IO.Pipes": "4.0.0",
+        "System.Reflection": "4.1.0",
```

Comment:
@agocke can you be more specific on what needs to be done?  This change is holding up RC bug fixes.  Need to get moving on it.